tools/circleci_dependencies.sh

```diff
@@ -23,6 +23,7 @@ if [[ "$CIRCLE_JOB" == "interactive_test" ]]; then
 	python -m pip install --progress-bar off vtk-9.0.20201117-cp39-cp39-linux_x86_64.whl
 	python -m pip install --progress-bar off https://github.com/pyvista/pyvista/zipball/main
 	python -m pip install --progress-bar off https://github.com/pyvista/pyvistaqt/zipball/main
+	python -m pip install --progress-bar off --upgrade --pre mne-qt-browser
```

Comment:
We don't need this for CircleCI. Eventually we should add it but let's do it later: #9942